README.md

```diff
@@ -231,14 +233,11 @@ To make `run-tnf-container.sh` use the newly built image, specify the custom TNF
 
 ## Building and running the standalone test executable
 
-Currently, all available tests are part of the "CNF Certification Test Suite" test suite, which serves as the entrypoint
-to run all test specs.  `CNF Certification 3.0` is not containerized, and involves pulling, building, then running the
-tests.
-
+Currently, all available tests are part of the "CNF Certification Test Suite" test suite, which serves as the entrypoint to run all test specs.
 By default, `test-network-function` emits results to `test-network-function/cnf-certification-tests_junit.xml`.
 
-The included default configuration is for running `generic` and `multus` suites on the trivial example at
-[cnf-certification-test-partner](https://github.com/test-network-function/cnf-certification-test-partner).  To configure for your
+The included default configuration is for running `diagnostic`  suite on the trivial example at
+[cnf-certification-test-partner](https://github.com/test-network-function/cnf-certification-test-partner). To configure for your
 own environment, please see [config.md](docs/config.md).
```

Comment:
this file no longer exists

Reply:
updated to point to configuration example in partner repo